### PR TITLE
feature(ui): override the default search react app from invenio-app-rdm

### PR DESCRIPTION
**Pull request by utnapischtim**
The look of the result entries should now be the same as on the frontpage.

The main difference from the default layout is that the access badge is not more a
badge. It is now in the footer on the right side. The main reason is that the
green color of the badge could lead to the wrong conclusion that if this record
has the access right "Open Acess" it would be a "Green Open Access" record.

NOTES:
It was necessary to import also some React classes into the components.js file
which were not really overriden. This was because it was not possible to import
it from invenio-app-rdm/search/components.js directly. This should be repaired
in some ways.

### CODE REVIEW

File: invenio_theme_tugraz/config.py [hunk 1]
```diff
@@ -94,3 +94,6 @@
 #    template="invenio_theme_tugraz/record_landing_page.html"
 # )
 """override the default record landing page"""
+
+SEARCH_UI_SEARCH_TEMPLATE = "invenio_theme_tugraz/search.html"
+"""override the default search page"""
```
Comment (mb-wali) [inline invenio_theme_tugraz/config.py:99]: Question:  From which module/package is this config variable?

**Review by mb-wali — COMMENTED** [review]
All looks good - I do have this suggestion:
what if we change the badge colors as below:

**From**
from

**To**
to
*colors: teal and grey*

**Question:** To be clear invenioRDM used different colors for access_rights, but in this case we will have the same color for all - is this fine? See below:
question

**invenioRDM access_right colors:**
invenioRDM version

**Comment by mb-wali** [comment]
**Merging with:**
search

**Comment by mb-wali** [comment]
The records landing page badge colors went crazy, have a look at the screenshot:

bu-landing page

**Comment by utnapischtim** [comment]
i close this PR. It is messed up with a wrong merge of the master into the branch. The new PR is #142

**Pull request closed without merging**